editor/src/clj/editor/atlas.clj

```diff
@@ -637,7 +637,8 @@
   (let [[width height] layout-size
         pages (group-by :page layout-rects)
         child-renderables (into [] (for [[page-index page-rects] pages] (produce-page-renderables aabb width height page-index page-rects gpu-texture)))]
-    {:aabb aabb
+    {:node-id _node-id
```

Comment:
I don't get why don't we add it... In all the other places we have `:node_id` but not here. I hope it's fine I added it